### PR TITLE
Small fixes

### PR DESCRIPTION
**Pull request by maryarbab**
**Is this a new addition or an update to an existing factor?**

- [ ] New
- [x] Update

**What category or categories does it affect?**

Vehicle, Road Transport, Air Transport

**What?**

Some small fixes to descriptions and lca_activity.

**Why?**

Clarifies. some EPA lca_activity. Changed "Vehicle" category name to "Vehicles".

### CODE REVIEW

File: OpenEmissionFactorsDB.csv [hunk 1]
```diff
@@ -2380,297 +2380,297 @@ Transport,Freight Services,transport_services-type_downstream_transportation_dis
...
+Transport,Vehicles,commercial_vehicle-vehicle_type_lcv-fuel_source_bev-engine_size_large-vehicle_age_post_2015-vehicle_weight_na,Battery EV LCV (large) post-2015,km,,0.03107538,,,,,,,,MfE,2020,NZ,2021/09/10,Emission intensity of commercial large battery EV. Light commercial vehicles. No upstream or downstream LCA activity is included. Post-2015 fleet.  Published by the Ministry for the Environment of New Zealand in 2020. Retrieved from 2020 Emission Factors Flat File.,https://environment.govt.nz/publications/measuring-emissions-detailed-guide-2020/,electricity_consumption
```
Comment (AnastasiaLobanova) [inline OpenEmissionFactorsDB.csv]: Is there (and in other NZ efs below) an extra white space before the "Published" in the description?  Just a small thing

File: OpenEmissionFactorsDB.csv [hunk 2]
```diff
@@ -2380,297 +2380,297 @@ Transport,Freight Services,transport_services-type_downstream_transportation_dis
...
+Transport,Vehicles,commercial_vehicle-vehicle_type_hgv-fuel_source_diesel-engine_size_na-vehicle_age_post_2015-vehicle_weight_gt_30t,Diesel HGV (>30t) post-2015,km,,1.49227543,,,,,,,,MfE,2020,NZ,2021/09/10,Emission intensity of heavy goods vehicle. Diesel. Above 12t and 30t. Heavy goods vehicles. No upstream or downstream LCA activity is included. Post-2015 fleet.  Published by the Ministry for the Environment of New Zealand in 2020. Retrieved from 2020 Emission Factors Flat File.,https://environment.govt.nz/publications/measuring-emissions-detailed-guide-2020/,fuel_combustion
+Transport,Vehicles,commercial_vehicle-vehicle_type_hgv-fuel_source_diesel-engine_size_na-vehicle_age_post_2015-vehicle_weight_gt_10t_lt_12t,Diesel HGV (10 - 12t) post-2015,km,,0.70446052,,,,,,,,MfE,2020,NZ,2021/09/10,Emission intensity of heavy goods vehicle. Diesel. Between 10t and 12t. Heavy goods vehicles. No upstream or downstream LCA activity is included. Post-2015 fleet.  Published by the Ministry for the Environment of New Zealand in 2020. Retrieved from 2020 Emission Factors Flat File.,https://environment.govt.nz/publications/measuring-emissions-detailed-guide-2020/,fuel_combustion
```
Comment (AnastasiaLobanova) [inline OpenEmissionFactorsDB.csv]: Here the same comment as above?

**Review by AnastasiaLobanova — CHANGES_REQUESTED** [review]
Clean! 👍 

Just a small comment: some extra spaces in the descriptions of NZ efs before "Published".

**Review by patstirling — APPROVED** [review]
Looks good to me - suggest we do a find/replace of double spaces as a follow-up PR

**Review by AnastasiaLobanova — APPROVED** [review]

**Pull request merged**